src/components/Rockets.js

```diff
@@ -1,4 +1,16 @@
 import React from 'react';
+import { useSelector, shallowEqual } from 'react-redux';
+import Rocket from './Rocket';
+import '../css/Rockets.css';
+
+const Rockets = () => {
+  const rockets = useSelector((state) => state.rockets, shallowEqual);
```

Comment:
I actually took this out of my Missions component and placed it in the App.js so that there is a prop flow downwards and a state bubble upwards flow of data. All the states are retrieved in the app then functions affecting it are passed down to their respective elements

Reply:
@amiraabouhadid, I am done implementing these changes.
I am awaiting your review so I can move forward with other tasks.